File: app/src/main/java/com/teslyuk/androidtutorial_sqlitedatabase/util/Logger.java
package com.teslyuk.androidtutorial_sqlitedatabase.util;

import android.util.Log;

/**
 * Created by taras on 25.11.15.
 */
public class Logger {

    public static void i(String TAG, String message) {
        Log.i(TAG, message);
    }

    public static void e(String TAG, String message) {
        Log.e(TAG, message);
    }

    public static void d(String TAG, String message) {
        Log.d(TAG, message);
    }
}

File: app/src/main/java/com/teslyuk/androidtutorial_sqlitedatabase/util/Logger.kt
package com.teslyuk.androidtutorial_sqlitedatabase.util

import android.util.Log

/**
 * Created by taras on 25.11.15.
 */
object Logger {

    fun i(TAG: String, message: String) {
        Log.i(TAG, message)
    }

    fun e(TAG: String, message: String) {
        Log.e(TAG, message)
    }

    fun d(TAG: String, message: String) {
        Log.d(TAG, message)
    }
}
